### ISSUE TITLE
Including continuous endpoints in Citrus Run

### ISSUE THREAD

**Issue by SforR** [nolanlab/citrus#120]
Dear all,

I have succesfully used the User Interface program and I get great results.
However I am stuck with the next step. I want to include endpoints as continues values of the sampels but that is only possible when using the R script directly as mentioned in the "Getting started" protocol. 

Does anyone has experience with that? I saw that there is a specific R script for the continuous endpoint ("citrus.continuousModel.R")  but I don't know how to use this.

My first question is actually where and how to integrate the continues values. Am I right that this should be referring by the name "features" in the script? And should the format be similar as in the example CSV at the location: https://github.com/nolanlab/citrus/blob/master/inst/extdata/example2/citruskey.csv

I have no idea where to start. Can someone help me out with this? I am lacking some experience with R I suppose.

Best,

Sam

**Comment by SamGG** [comment]
Hi,
I never used it, but I dived a little bit in the code.
I think CSV of exemple5 is correct, other examples concern classification, ie building separated groups. Most of examples (1 to 4) concern classification, and `family` parameter is set by default to `classification`. I think you have to change it to `continuous`.
You might look for the keyword **continuous**, and you will lead to the following functions and help messages written in R code.
https://github.com/nolanlab/citrus/blob/master/R/citrus.full.R
https://github.com/nolanlab/citrus/blob/master/R/citrus.model.R
https://github.com/nolanlab/citrus/blob/master/R/citrus.continuousModel.R
If you build an example script, that would be great.
HTH

**Comment by SforR** [comment]
Hi HTH,

Thank you for your input and thinking along. Example 5 is indeed what I am looking for, so that could be a hint. I have searched for example5 in the scripts but couldn't find anything.
Now I will indeed search for "continuous".

Thanks again and I will keep you updated.

Sam

**Comment by SamGG** [comment]
By chance
https://github.com/nolanlab/citrus/blob/master/inst/citrus.test.survival.R
Best.

**Comment by SforR** [comment]
Samuel,

I played around with the R script based on the "runcitrus.R" script for the GUI output.
Here I have changed the family to continuous and add different values in labels as.factor (see attachment for an example script). 
 
When I run this script I see on the Y-ax the different values, however I don't know what the alghorithm exactly compares or calculates.

Best,

Sam


[Test script for continuous values.txt](https://github.com/nolanlab/citrus/files/2090315/Test.script.for.continuous.values.txt)
